### DISCUSSION TITLE
combined all modifier together ! -- just lil idea

### DISCUSSION THREAD

**Discussion by imousart** [tailwindlabs/tailwindcss#9053] [Ideas]
class=" flex active:{ bg-gray-900 outline-non } hover:{ bg-gray-600 shadow ... } w-full "

thats it .. like this i think its more clear and easy if can this happen :) 

thx

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

**Reply by danirdd92** [reply]
So basically, it was scrapped because it adds up even more classes to the css file?

What if it was a step converter during JIT? I mean just transpile it to "vanilla" tailwind css.
